pkg/controller/kieapp/defaults/upgrade_test.go

```diff
@@ -94,8 +94,8 @@ func TestCheckProductUpgrade(t *testing.T) {
 	assert.True(t, micro)
 
 	diffs = configDiffs(getConfigVersionLists(cr.Status.Applied.Version, constants.CurrentVersion))
-	// assert.NotEmpty(t, diffs)
-	assert.Empty(t, diffs)
+	assert.NotEmpty(t, diffs)
+	//assert.Empty(t, diffs)
```

Comment:
these upgrade test changes may be unnecessary in this branch because you still need to cherry-pick the commit from the other 7.11.1 PR so that those 7.11.1 configs make it into the main branch as well. once that cherry-picking occurs... there may not be any diffs between version configs.

Reply:
ok, reverted